### ISSUE TITLE
Associated files have English names in Windows Explorer, even though the system doesn’t use English locale

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [sumatrapdfreader/sumatrapdf#3323]
After installing current version (3.4.6), I saw the PDF files in Windows Explorer named “PDF File” for file type, although I used zh-CN locale.

[![1.png](https://i.postimg.cc/13ctfM0R/1.png)](https://postimg.cc/pmrxSf13)

As my investigation, this name is registered in HKEY_CLASSES_ROOT\SumatraPDF.XXX paths, making all file type name hardcoded in English, this would affect users who used non-English version of Windows. If I delete the _(Default)_ registry item, the system will display Chinese name for me.

[![2.png](https://i.postimg.cc/6QMph31D/2.png)](https://postimg.cc/zLRNzJNj)
[![3.png](https://i.postimg.cc/zB3X2rs3/3.png)](https://postimg.cc/R65BqjVz)
(The associated icons were modified manually after seeing #3074.)

So I suggest removing such operation in installer.

**Comment by vlakoff** [comment]
See [Programmatic Identifiers (ProgID)](https://learn.microsoft.com/en-us/windows/win32/shell/fa-progids), in particular the `(Default)` and `FriendlyTypeName` elements.

The recommended method seems to be to use the `FriendlyTypeName` element, but I don't know what would be the index of the "indirect string", nor if the SumatraPDF.exe's resources contain the strings we want.

**Comment by KrasnayaPloshchad** [comment]
@kjk What happened if you try Polish version of Windows?

**Comment by vlakoff** [comment]
Indeed, using `FriendlyTypeName` with indirect localized strings from the binary's resources would be overkill here.

Letting Windows automatically generate the localized name via the fallback mechanism is much simpler and achieves exactly what we want.
